Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
I'm not a big fan of `@covers` on method.

AFAIK, is you write `@covers Some::thing` and `Some::that` is called, it would not be covered even if it's tested.

Maybe `@covers` on the class?